{{cookiecutter.repo_name}}/requirements/test.txt

```diff
@@ -1,12 +1,13 @@
 # Packages required for testing
 -r base.txt
 
-coverage
-django-dynamic-fixture
-django-nose
-mock
-nose-ignore-docstring
-pep8
-pylint
+coverage == 4.0.2
```

Comment:
Makes sense. I set versions because a recent pylint build was breaking the tests.